.scalafmt.conf

```diff
@@ -1,9 +1,14 @@
+# scalafmt configuration.
 style = defaultWithAlign
 
-danglingParentheses        = true
-indentOperator             = spray
-maxColumn                  = 100
-project.excludeFilters     = [".*\\.sbt"]
-rewrite.rules              = [RedundantBraces, RedundantParens, SortImports]
-spaces.inImportCurlyBraces = true
-unindentTopLevelOperators  = true
\ No newline at end of file
+maxColumn                     = 100
+rewrite.rules                 = [RedundantBraces, RedundantParens, SortImports]
+indentOperator                = spray
+importSelectors               = singleLine
+spaces.inImportCurlyBraces    = true
+unindentTopLevelOperators     = true
+optIn.annotationNewlines      = true
+align.openParenCallSite       = false
+align.openParenDefnSite       = false
+continuationIndent.defnSite   = 2
```

Comment:
If you don't mind I'd rather to stick to the default value for this parameter, which is `4`. That is the default scala style recommendation for continuation indentation.

Reply:
You're right! I'm going to fix that right away.